core/classes/Database/PhinxAdapter.php

```diff
@@ -34,21 +33,7 @@ static function ($file_name) {
             foreach ($missing as $missing_migration) {
                 echo " - $missing_migration" . '<br>';
             }
-        }
-
-        // Something went wonky, either they've deleted migration files,
```

Comment:
The situation could still arise in development where migrations have been executed and stored in `nl2_phinxlog` - but I can't see a way that alerting the developer that this happened is beneficial. All the times it has happened to me so far I just go delete the row(s) that shouldn't exist on the current branch and then refresh the page - a slight annoyance.

For example:
- Checkout Member list branch, run the migration to create the member lists table
- Checkout develop, see an error that a migration exists in `nl2_phinxlog` which does not have a related migration file on the branch
I don't know in what case this would be a helpful message to know, since it can't be harmful?

> in a dev environment, I check out from a branch with new migrations back to develop, would I see a descriptive error message?

Not with this code removed, do you think it would be helpful information to know? I'd be fine to add this check back, but so far it's just been an added annoyance when working on multiple features.

Reply:
If removing this message means the website would load normally despite extra migrations being run, then I'm all for it.

If, and only if, we'd get a fatal error instead then I think we should keep it?

Reply:
> Checkout develop, see an error that a migration exists in nl2_phinxlog which does not have a related migration file on the branch
I don't know in what case this would be a helpful message to know, since it can't be harmful?

The warning makes sense to me. If the migration file does not exist on a branch, that means it still expects the "old" database format. If the migration has already been run (is in the phinxlog table), the database is in the "new" format.

Without this check, you may run into random database errors

Reply:
Makes sense, reverted. Thanks for the input